RWS-Submission_Guidelines.md

```diff
@@ -220,7 +226,8 @@ The /.well-known/related-website-set.json file for set members must follow the s
   "required": ["primary"]
 }
 ```
-## Subset-level technical validation ##
+
+### Subset-level technical validation
 
 Additionally, more granular technical checks will also run on GitHub for service domains and ccTLD variants in the submissions. 
 
```

Comment:
If you feel like doing more formatting cleanup, we could convert the `<ul>` below to use markdown syntax (`*`).